nexus/tests/integration_tests/endpoints.rs

```diff
@@ -1474,14 +1474,14 @@ lazy_static! {
         /* Sagas */
 
         VerifyEndpoint {
-            url: "/system/sagas",
+            url: "v1/system/sagas",
```

Comment:
```suggestion
            url: "/v1/system/sagas",
```

Reply:
Once you update this and the one below it you'll need to run expectorate for the authz test to add the old API endpoints to the authz exceptions file. The command is

```
EXPECTORATE=overwrite cargo test test_unauthorized_coverage
```